### PR TITLE
Adding account override command line flag

### PR DESCRIPTION
**Pull request by shraykay**
if you type `k8s-pipeline create test-pipeline --linear --override=int-k8s:ops` any stage that has `int-k8s` listed becomes `ops`.

### CODE REVIEW

File: cmd/k8s-pipeliner/main.go [hunk 1]
```diff
@@ -42,6 +43,10 @@ func main() {
 					Name:  "timeout",
 					Usage: "override the default 72 hour timeout (unit: int)",
 				},
+				cli.StringSliceFlag{
+					Name:  "override",
+					Usage: "override an environment with a different environment (example --override=int-k8s:int), --override=<old env>:<new env>, must be separated by colon",
```
Comment (mhamrah) [inline cmd/k8s-pipeliner/main.go]: Not 100% on the param options, but why stringslice as opposed to string? Should this be a comma-delimited list of pairs?

**Comment by mhamrah** [comment]
This looks fine, my only comment is larger than the PR which is why isn't this a 'shadow' so we can duplicate one env to another without modifying the pipeline.yaml file (as opposed to override)?

File: cmd/k8s-pipeliner/main.go (hunk 1 above)
Comment (shraykay) [reply cmd/k8s-pipeliner/main.go]: --override=cat:dog --override=int:ops etc etc I'm open to either way, makes no difference to me

Comment (shraykay) [reply cmd/k8s-pipeliner/main.go]: your version would be: `--override=cat:dog,int:ops`

Comment (mhamrah) [reply cmd/k8s-pipeliner/main.go]: makes sense. drop the commas in the usage and add (supports multiple entries)

**Comment by shraykay** [comment]
@mhamrah this is shadow in estuary, this is just the k8s-pipeliner side so the feature is complete.

**Pull request merged**